src/app/store-stats/CharacterStats.m.scss

```diff
@@ -2,80 +2,58 @@
 
 @layer base {
   :global(.stat) {
-    flex: 1;
     display: flex;
     flex-direction: row;
     align-items: center;
-    margin-right: 2px;
     line-height: 16px;
     white-space: nowrap;
-    &:last-child {
-      margin-right: 0;
+    font-size: 11px;
+    gap: 2px;
+    &:global(.boostedValue) {
```

Comment:
This is another change - now boosted stats will once again properly be highlighted on the loadouts page.